apps/codeforafrica/src/lib/index.js

```diff
@@ -1334,3 +1333,15 @@ export async function getPageStaticProps(params) {
       return { notFound: true };
   }
 }
+
+export async function getGhostCMSStaticPaths(primaryTag, options) {
```

Comment:
```suggestion
export async function getPageStaticPaths(primaryTag, options) {
```
There shouldn't be any need to include `GhostCMS` in the name (similar to `getPageStaticProps`